src/Diagnostics/graph_diagnostic.jl

```diff
@@ -0,0 +1,59 @@
+"""
+    graph_diagnostic.jl
+
+"""
+
+using DataFrames, FileIO
+
+include("diagnostic_vars.jl")
+
+function usage()
+    println("""
+Usage:
+    julia diagnostic_vars.jl <diagnostic_file.jld2> <diagnostic_name>""")
```

Comment:
should this be 
`julia graph_diagnostics.jl <diagnostic_file.jld2> <diagnostic_name>""")` ?

Reply:
Oops. Yes. Will fix.